rpa-manager/modules/ROOT/pages/processautomation-prepare-project-application.adoc

```diff
@@ -0,0 +1,111 @@
+= Manage Applications
+
+Process applications are third-party application installations that are required for the execution of a process. 
+
+Manage a list of all applications and their downtimes.
+
+Create a new entry for each instance of an application, for example `FinWiz - Mythical University - Computer 1 - Tokyo` and `FinWiz - Mythical University - Computer 42 - San Francisco`.
+
+Applications are not available during down-times or while deactivated. 
+
+To avoid confusion, append the name of the time zone to the name of the application or the downtime. For example, name an application `Words - Tokyo` and a downtime `Electrical Power Outage Tokyo`.
+
+In the *Process Management* view, a member of the project team xref:myrpa-start.adoc#required-process-applications[assigns required applications to a process] at process automation creation or phase transition.
+
+In the *Bot Management* module, an administrator xref:botmanagement-manage.adoc#bot-assign-servicetimes-applications[assigns required applications to bots] that can access them. 
+
+A member of the process team can deploy a process on a bot that has assigned the required applications. If an application becomes non-available, the bot completes running processes using that application and restarts them when the downtime or deactivation period has ended.
+
+== Before You Begin
+
+* Ask an administrator to xref:usermanagement-manage.adoc#assign-privileges-to-a-user[assign you the needed privileges]:
++
+[cols="1,2"]
+|===
+|*Action* |*Required Privileges*
+
+|Open the *Process Automation* module
+|Process Automation Open
+
+|Create, change, and delete process applications
+|Application Create, Application Edit, Application Delete
+
+|===
+* For each application, note the timezone of the computer that it is installed on.
+
+== View All Applications
+
+To view all existing applications for automation projects, open the *Process Applications* view of the *Process Automation* module.
+
+== Create a Process Application
+
+Create one entry for each application instance required for the execution of processes.
+
+To create a process application:
+
+. Open the *Process Applications* view of the *Process Automation* module.
+. Click *Create*.
+. [[form-create-process-applications]] Complete the *Create a new application* form:
++
+* *Time Zone*:
++
+Time zone for defining downtimes.
++
+Specify a downtime in the time zone in which it occurs. If the bot that you assign the downtime to is installed on a computer that uses a different time zone, the downtime interval is converted to the bot’s time zone. If you assign the application to multiple bots in different time zones, all bots pause executing processes that use that application simultaneously. The time intervals might change, depending on daylight saving time.
```

Comment:
What does "it" refer to here?

Reply:
to downtime

Reply:
Is that ambiguous?  A time zone cannot occur.

Reply:
I think it's fine. Ignore previous comment :)